docs/whats_new/v0.7.0.md

```diff
@@ -0,0 +1,145 @@
+
+This version supports Python versions from 3.8 to 3.10 and dropped the support of Python 3.7. 
+
+This version is also available on [conda-forge](https://anaconda.org/conda-forge/autogluon). Kudos to following members for making this happen:
+  * @giswqs for leading the entire effort and being a 1-man army driving this forward.
+  * @h-vetinari for providing excellent advice for working with conda-forge and some truly exceptional feedback.
+  * @arturdaraujo, @PertuyF, @ngam and @priyanga24 for their encouragement, suggestions, and feedback.
+  * The conda-forge team for their prompt and effective reviews of our (many) PRs.
+  * @gradientsky for testing M1 support during the early stages.
+  * @sxjscience, @zhiqiangdon, @canerturkmen, and @shchur for helping upgrade our downstream dependency versions to be compatible with conda.
+  * Everyone else who has supported this process either directly or indirectly.
+
+# Changes
+
+## Exploratory Data Analysis Tools
+
+We are happy to announce AutoGluon Exploratory Data Analysis (EDA) toolkit. Starting with v0.7, AutoGluon now can analyze and visualize different aspects of data and models. We invite you to explore the following tutorials: [Quick Fit](https://auto.gluon.ai/dev/tutorials/stable/eda-auto-quick-fit.html), [Dataset Overview](https://auto.gluon.ai/stable/tutorials/eda/eda-auto-dataset-overview.html), [Target Variable Analysis](https://auto.gluon.ai/stable/tutorials/eda/eda-auto-target-analysis.html), [Covariate Shift Analysis](https://auto.gluon.ai/stable/tutorials/eda/eda-auto-covariate-shift.html). Other materials can be found in [EDA Section](https://auto.gluon.ai/stable/tutorials/eda/index.html) of the website.
+
+
+## AutoMM
+
+AutoGluon MultiModal (a.k.a AutoMM) supports three new features: 1) document classification; 2) named entity recognition
+for Chinese language; 3) few shot learning with SVM  
+
+Meanwhile, we removed `autogluon.text` and `autogluon.vision` as these features are supported in `autogluon.multimodal`
+
+### New features
+
+- Document Classification
+  - Add scanned document classification (experimental).
+  - Customers can train models for scanned document classification in a few lines of codes
+  - See [tutorials](https://auto.gluon.ai/stable/tutorials/multimodal/document/document_classification.html)
+  - Contributors and commits: @cheungdaven (#2765, #2826, #2833, #2928)
+- NER for Chinese Language
+  - Support Chinese named entity recognition
+  - See [tutorials](https://auto.gluon.ai/stable/tutorials/multimodal/document/document_classification.html)
+  - Contributors and commits:  @cheungdaven (#2676, #2709)
+- Few Shot Learning with SVM
+  - Improved few shot learning by adding SVM support
+  - See [tutorials](https://auto.gluon.ai/stable/tutorials/multimodal/advanced_topics/few_shot_learning.html)
+  - Contributors and commits: @yongxinw (#2850)
+
+### Other Enhancements
```

Comment:
Missing inference speedup highlight?

Reply:
We didn't have tensorrt support merged yet. Meanwhile, I didn't see much significant speedup with onnx export functionality.